<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Progetto</class>
 <widget class="QMainWindow" name="Progetto">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Progetto</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>110</y>
      <width>251</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="realSum">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>92</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>realSum</string>
    </property>
   </widget>
   <widget class="QPushButton" name="realSottr">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>0</y>
      <width>92</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>realsottr</string>
    </property>
   </widget>
   <widget class="QPushButton" name="realPer">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>0</y>
      <width>92</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>realPer</string>
    </property>
   </widget>
   <widget class="QPushButton" name="realDiv">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>0</y>
      <width>92</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>realDiv</string>
    </property>
   </widget>
   <widget class="QPushButton" name="complexDiv">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>40</y>
      <width>92</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>complexDiv</string>
    </property>
   </widget>
   <widget class="QPushButton" name="complexSottr">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>40</y>
      <width>92</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>complexSottr</string>
    </property>
   </widget>
   <widget class="QPushButton" name="complexPer">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>40</y>
      <width>92</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>complexPer</string>
    </property>
   </widget>
   <widget class="QPushButton" name="complexSum">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>92</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>complexSum</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>400</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
